src/components/Comment/Comment.tsx

```diff
@@ -130,6 +146,8 @@ export const Comment = ({
   );
   const [error, setError] = useState<string>();
 
+  const isCommentBlocked = comment.status === "blocked";
+
```

Comment:
I reckon we could probably inline `comment.status === "blocked" ? ()` - fewer abstractions is always nice and it solves the !isTrue confusion